03_letters/saurabhvyas9/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func letters(s string) map[rune]int {
+	frequencyLetters := make(map[rune]int)
+	for _, str := range s {
```

Comment:
`str` is a poor choice of name for this - it is not a string and pretty much everyone would expect a variable called `str` to be a string. Just `r` would be sufficient (short for rune) and that makes a lot of sense for ranging over a variable called `s` (which I would expect to be a string).

Reply:
Understood - appropriate variable names.